packages/manager/apps/public-cloud/src/assets/theme/_sidebar.less

```diff
@@ -203,4 +203,28 @@ cloud-sidebar { /* stylelint-disable-line */
 
 #sidebar-menu .oui-sidebar-menu {
   width: 100%;
+
+  .sidebar-menu-orphan-item {
+    border-top: #89d7f8 2px solid;
```

Comment:
More convenient to store variables into `_variables.less`

Reply:
I don't know if that makes sense as it is meant to only be used here

Reply:
Even if the style is in place for a short period of time, keeping it organized feel more convenient.

Otherwise adding a comment which describe that it's a temporary style with a `TODO: remove as soon as...` for instance.

Is that make sens to you?